src/main/java/edu/oakland/production/database/DatabaseGisManagerStub.java

```diff
@@ -0,0 +1,23 @@
+package edu.oakland.production.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+
+public class DatabaseGisManagerStub implements DatabaseGisManager {
+
+  public String passNextSatRequest(String currentSat) {
+    return "";
```

Comment:
Please make sure that this passes out the `currentSat` value, otherwise you don't really learn anything from this test.